AppInspector/Commands/AnalyzeCommand.cs

```diff
@@ -333,17 +344,17 @@ void ProcessAndAddToMetadata(FileEntry file)
                     }
                     else
                     {
-                        _ = _metaDataHelper?.FileExtensions.TryAdd(Path.GetExtension(file.FullPath).Replace('.', ' ').TrimStart(), 0);
```

Comment:
.ToLower().Replace('.', ' ').TrimStart()?

Reply:
This is a removed line.